billing/Earthfile

```
@@ -0,0 +1,11 @@
+VERSION 0.6
+
+FROM alpine/git:2.36.3
+
+# Estimates monthly active users of a GIT remote repo, based on past commits
+active-users:
+  ARG repoUrl
+  DO ../+REQUIRE --argName repoUrl --argValue $repoUrl
```

You probably don't need `REQUIRE` - there's an earthly feature for that

```suggestion
  ARG --required repoUrl
```